roles/openshift_logging_elasticsearch/library/openshift_logging_elasticsearch_facts.py

```diff
@@ -0,0 +1,351 @@
+'''
```

Comment:
Do we really need this duplicated from the logging role?  If we have fact gathering for the entire logging role why do we  this here?